src/utils/api.ts

```diff
@@ -0,0 +1,20 @@
+export const sendContactForm = async (values: {
+  name: string
+  email: string
+  message: string
+  reference: string
+  referenceOther: string
+}) => {
+  const body = {
+    'form-name': 'clublive-contact',
+    ...values,
+  }
+
+  return fetch('/api/contact', {
```

Comment:
this is wrong, it needs to be:
```suggestion
  return fetch('/', {
```
Compare to this: https://github.com/julisch94/clublive-website/blob/42d3b4ca7e18951115d15b66054f6676861dc6dc/src/components/ContactForm.vue#L141-L145